docs/reference/mapping/types/keyword.asciidoc

```diff
@@ -129,3 +129,5 @@ The following parameters are accepted by `keyword` fields:
 include::constant-keyword.asciidoc[]
 
 include::wildcard.asciidoc[]
+
+include::version.asciidoc[]
```

Comment:
@cbuescher  I am curious why you considered `version` to be a part of keyword family? 

I would rather say that it should be a part of `structured` or `other` category?

Also curious what @jtibshirani  thinks as she worked on this organization.

Reply:
I put the docs here because we discussed the desired compatibility with the `keyword` field for the new field type a lot since users currently mostly seem to use `keyword` to indec these version constants. The `version` field should more or less behave like a keyword field, just with specialized semantics and ordering. I'm happy to move the docs anywhere else if there is another option that finds consensus.

Reply:
thanks for the explanation, let's see what @jtibshirani thinks of this categorization

Reply:
Thanks for looping me in, I was also imagining it would be listed under 'Structured'.

We recently added this concept of a 'field type family', which is a group of field types that have identical search behavior, but present different trade-offs (for example in terms of query speed). They are all present as the family type in field caps. The `version` field doesn't fit this description -- it has different search behavior on purpose, and doesn't present itself as a `keyword` in field caps.

Reply:
Thanks @jtibshirani I moved the docs in https://github.com/elastic/elasticsearch/pull/59773/commits/ccebd6f470c99c824a84211ed122ba10f1f9629c